pyserini/search/__main__.py

```diff
@@ -16,30 +16,30 @@
 
 import argparse
 import re
+from tqdm import tqdm
 from pyserini.search.pysearch import get_topics, SimpleSearcher
 from pyserini.search.reranker import ClassifierType, PseudoRelevanceClassifierReranker
 
-
 parser = argparse.ArgumentParser(description='Create a input schema')
-parser.add_argument('-index', metavar='path', required=True,
+parser.add_argument('-index', type=str, metavar='path', required=True,
                     help='the path to workspace')
```

Comment:
path to index?

also, we're using line width of 100, so some of these can be on a single line.

Reply:
```
parser.add_argument('-indexpath', type=str, metavar='path',
                    required=True, help='the path to lucene index')
```

Changed my editor to line width of 100.